main.go

```diff
@@ -65,6 +66,8 @@ var (
 	storageDirty          = flag.Bool("storage.local.dirty", false, "If set, the local storage layer will perform crash recovery even if the last shutdown appears to be clean.")
 	storagePedanticChecks = flag.Bool("storage.local.pedantic-checks", false, "If set, a crash recovery will perform checks on each series file. This might take a very long time.")
 
+	pathPrefix = flag.String("web.path-prefix", "/", "Prefix for all paths, defaults to /")
```

Comment:
The default will be displayed automatically in the help output, so I'd omit that and just write `Prefix for all web paths.`.